lms/djangoapps/program_enrollments/tests.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""
+ProgramEnrollment Tests
+"""
+from __future__ import unicode_literals
```

Comment:
Why is this file needed? We usually create a `tests` folder instead of a `tests.py` file.

Reply:
Sorry this is part of the auto-generated boilerplate. Will remove